app/webpacker/components/Panel/pages/EditPersonPage/EditPersonForm.jsx

```diff
@@ -51,6 +51,16 @@ export default function EditPersonForm({ wcaId, onSuccess, showDestroyButton = f
     setEditedUserDetails((prev) => ({ ...prev, [formName]: value }));
   };
 
+  const handleDobChange = (date) => handleFormChange(null, {
+    name: 'dob',
+    value: date,
+  });
+
+  const handleCountryChange = (e, { value }) => handleFormChange(e, {
+    name: 'representing',
+    value,
+  });
```

Comment:
I think this extra callback is redundant now. If you propagate the `name` to the country selector directly as prop, it will be part of the `data` payload (that is now "nicely" propagated since your last commit)